gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/modules/image-resolution/data/image-overrides.list.ts

```diff
@@ -0,0 +1,490 @@
+import { boosterSelectionImage, randomFoilPackImage } from "../types/image-location.constants";
```

Comment:
again, should have been a JSON, but it is what it is.

Also gave them better naming conventions, and everything else is broken down into files that are essentially sequentially loaded. Yea, it should all be an API, but still, we got what we got.